python/pyproject.toml

```diff
@@ -2,8 +2,8 @@
 requires = ["setuptools", "wheel"]
 
 [project]
-name = "python-aiconfig"
-version = "1.1.9"
+name = "python-aiconfig-test"
+version = "1.1.25"
```

Comment:
For future, we don't need to keep changing this. We can have two separate repos and simply when we do pip install we explicitly pull from our test repo:

`pip3 install --index-url https://test.pypi.org/project/python-aiconfig-test/`

^We can use `python-aiconfig` instead of `python-aiconfig-test`

I supposed if we are going to update the the version number to match pytest anyways, it's not that much different to change package name so that's fine too
cc @jonathanlastmileai